packages/cli/src/commands/test.ts

```diff
@@ -89,14 +89,15 @@ export default class Test extends AuthCommand {
       'env-file': envFile,
       list,
       timeout,
-      verbose,
+      verbose: verboseFlag,
     } = flags
     const cwd = process.cwd()
     const filePatterns = argv as string[]
 
     const testEnvVars = await getEnvs(envFile, env)
     const { config: checklyConfig, constructs: checklyConfigConstructs } = await loadChecklyConfig(cwd)
     const location = await this.prepareRunLocation(checklyConfig.cli, { runLocation, privateRunLocation })
```

Comment:
Taking a closer look, I'm a bit confused by the error that I reported in https://github.com/checkly/checkly-cli/issues/498

The exact error message was
```
TypeError: Cannot read properties of undefined (reading 'cli')
    at Test.run (.../node_modules/@checkly/cli/src/commands/test.ts:98:66)
```

That must mean that `checklyConfig` here was undefined - not that `checklyConfig` was missing the `cli` property.

Reply:
Or is this issue fixed by adding the default `cliFlags` value of `{}` for `prepareRunLocation`?

Reply:
Were you able to reproduce the issue by removing the `cli` section?

Do you have any thoughts on how we can avoid a regression here? Will the TS compiler help us at all? Should we add a test case?

Reply:
Actually, you are right. This is about config being null or undefined. I need to check the value returned from the config files. This PR doesn't close that issue but we can still merge it

Reply:
Sounds good 👍 

Could you please remove the `Resolves https://github.com/checkly/checkly-cli/issues/498` from the PR description? Otherwise I think that the issue will be automatically closed and we might lose track of it.

Reply:
Makes sense